twitter/Module.md

```diff
@@ -0,0 +1,52 @@
+## Overview
+Connects to Twitter using Ballerina.
+
+The Twitter API can be used to programmatically retrieve data and engage with the conversation on Twitter. When accessing Twitter through Ballerina Twitter Connector, It is required to register a developer application. By default, applications can only access public information on Twitter. Certain operations, such as those responsible for sending or receiving direct Messages and posting a tweet, require additional permissions from user before access user information. These permissions are not granted by default; you choose on a per-application basis whether to provide this access, and can control all the applications authorized on your account.
```

Comment:
Let's rephrase this as: "The Twitter API can be used to programmatically retrieve data and to engage in a conversation on Twitter. When accessing Twitter through Ballerina Twitter Connector, it is required to register a developer application. By default, applications can only access public information on Twitter. Certain operations, such as those responsible for sending or receiving direct messages and posting a Tweet, require additional permissions from the user before accessing user information. These permissions are not granted by default, and you can choose on a per-application basis if to provide access. Furthermore, you can control all the applications authorized on your account."

Reply:
Added as mentioned